### ISSUE TITLE
 meta-updater/classes/sota_none.bbclass spits out too many warning messages

### ISSUE THREAD

**Issue by dl9pf** [advancedtelematic/meta-updater#69]
As this warning message is in the global namespace any task/action will print this out. Over and over.
Also this means we slow-down processing as this is always executed and not just once.

**Comment by dl9pf** [comment]
To reproduce just build for an unsupported board.

**Comment by cajun-rat** [comment]
Good point. Easiest fix is likely to nuke the warning altogether, but maybe we can persuade it to print only once.

**Comment by dl9pf** [comment]
Ping?

**Comment by cajun-rat** [comment]
Oops, @OYTIS, Am I right in thinking we can just nuke the whole block?

~~~
python __anonymous() {
	bb.warn("SOTA functionality is not yet supported for your machine")
}
~~~

**Comment by dl9pf** [comment]
Well, we should somehow warn the user once, but right now this message is printed not once but for *every* task. Thus like 10000 times.

**Comment by cajun-rat** [comment]
Fixed in #93 and #94 , thank you @dl9pf !
